onnxscript/function_libs/tools/torch_aten/generate_aten_signatures.py

```diff
@@ -237,7 +237,7 @@ def create_signature(func: torchgen.model.NativeFunction) -> cg.FunctionDef:
         *py_args,
         return_type=create_return_type(func.func.returns),
         body=[
-            cg.ThunkStmt(f"# {func.func}"),
+            cg.ThunkExpr(f'"""{func.func}"""'),
```

Comment:
I thought the CI typing issue is with the next line (Raise not being Stmt)?

Reply:
Yeah... The thing should have been an expression but the body was typed to only accept statements. I changed it back.